how-to.md

```diff
@@ -121,3 +121,7 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
   * **Tenants** > **Repos** > select dropdown at the top of the page > **Manage repositories** > **...** (next to the tenant you want to remove) > **Delete** > follow the on-screen instructions to delete the repository
   * **Pipelines** > **...** (next to the tenant you want to remove) > follow the on-screen instructions to delete the pipeline
   * **Environments** (on the left pane under **Pipelines**) > select the tenant you want to remove > **...** > **Delete** > **Delete**
+
+## Change user used for delegated authentication
+
+* To change the user you are using for delegated authentication, navigate to [Simeon](https://app.simeoncloud.com/install) > toggle off **New Tenant** > select the tenant under Display name > INSTALL > follow the steps on the screen > SYNC NOW > when the status of the tenant Sync is **Pending Authentication**, authenticate with a different user.
```

Comment:
For consistency with the other how tos, get rid of "to change the user you are using for delegated authentication"

Reply:
resolved